fpmsyncd/fpmlink.cpp

```diff
@@ -9,6 +9,86 @@
 using namespace swss;
 using namespace std;
 
+void netlink_parse_rtattr(struct rtattr **tb, int max, struct rtattr *rta,
+        int len)
+{
+    while (RTA_OK(rta, len)) 
+    {
+        if (rta->rta_type <= max)
+            tb[rta->rta_type] = rta;
+        rta = RTA_NEXT(rta, len);
+    }
+}
+
+bool FpmLink::isRawProcessing(struct nlmsghdr *h)
+{
+    int len;
+    short encap_type = 0;
+    struct rtmsg *rtm;
+    struct rtattr *tb[RTA_MAX + 1];
+
+    rtm = (struct rtmsg *)NLMSG_DATA(h);
+
+    if (h->nlmsg_type != RTM_NEWROUTE
+        && h->nlmsg_type != RTM_DELROUTE)
+        return 0;
+
+    len = (int)(h->nlmsg_len - NLMSG_LENGTH(sizeof(struct rtmsg)));
+    if (len < 0) 
+    {
+        return 0;
+    }
+
+    memset(tb, 0, sizeof(tb));
+    netlink_parse_rtattr(tb, RTA_MAX, RTM_RTA(rtm), len);
+
+    if (!tb[RTA_MULTIPATH])
+    {
+        if (tb[RTA_ENCAP_TYPE])
+            encap_type = *(short *)RTA_DATA(tb[RTA_ENCAP_TYPE]);
+    }
+    else
+    {
+        /* This is a multipath route */
+        int len;            
+        struct rtnexthop *rtnh = (struct rtnexthop *)RTA_DATA(tb[RTA_MULTIPATH]);
+        len = (int)RTA_PAYLOAD(tb[RTA_MULTIPATH]);
+        struct rtattr *subtb[RTA_MAX + 1];
+        
+        for (;;) 
+        {
+            if (len < (int)sizeof(*rtnh)
```

Comment:
Suggest to have it in a single line and as a general comment, use open/close braces for `if`

Reply:
Done as suggested